src/data/moves/move-attrs/compare-weight-power-attr.ts

```diff
@@ -18,24 +18,16 @@ export class CompareWeightPowerAttr extends VariablePowerAttr {
       return false;
     }
 
-    const relativeWeight = (targetWeight / userWeight) * 100;
-
-    switch (true) {
-      case relativeWeight < 20.01:
-        power.value = 120;
-        break;
-      case relativeWeight < 25.01:
-        power.value = 100;
-        break;
-      case relativeWeight < 33.35:
-        power.value = 80;
-        break;
-      case relativeWeight < 50.01:
-        power.value = 60;
-        break;
-      default:
-        power.value = 40;
-        break;
+    if (userWeight >= targetWeight * 5) {
+      power.value = 120;
+    } else if (userWeight >= targetWeight * 4) {
+      power.value = 100;
+    } else if (userWeight >= targetWeight * 3) {
+      power.value = 80;
+    } else if (userWeight >= targetWeight * 2) {
+      power.value = 60;
+    } else {
+      power.value = 40;
```

Comment:
```suggestion
    power.value = (clamp(Math.floor(userWeight / targetWeight), 1, 5) + 1) * 20;
```
May need a double-check, but I think this is equivalent.

Reply:
I'll write a test for it 👍 
Just gotta find some pokemon with the right weights...

Reply:
Oh, this also should have a way to handle when `targetWeight` is 0 (the `if` block before this doesn't cover divide by 0 issues anymore)